doc/chapters/api/index.rst

```diff
@@ -18,6 +18,13 @@ object path and interface.
 * Object Path: /com/pelagicore/SoftwareContainerAgent
 * Interface: com.pelagicore.SoftwareContainerAgent
 
+Prerequisities
+--------------
+The prerequisities for each method describe the absolute minimum needed for a call to that method to
```

Comment:
I would rephrase it a bit.
"The prerequisites for each method specifies the mandatory parameters necessary for a successful method call"
Also I'm not a fan of the next sentence. I have to read it many times to understand it. Perhaps rephrase that also.
E.g. "For most calls the only prerequisite is a successfully created container, which is the same as getting a container ID returned from the Create method"